src/components/CustomInput/type.ts

```diff
@@ -0,0 +1,12 @@
+import React, { MutableRefObject } from 'react';
+import { TextInput } from 'react-native';
+
+export type CustomInputPropsType = {
+  onChangeText?: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
let's fix all types to our type convention

all your optional type should be at the end, and separate them from obviously prop with enter line

example 
type YourType = {
prop1: stirngs, 
prop2: stirng,

prop3?: stirng,
prop4?: stting,
}

Reply:
ok